app/cdap/components/hydrator/components/SidePanel/SidePanel.tsx

```diff
@@ -136,69 +176,79 @@ export const SidePanel = ({
 
   const renderPlugins = (plugins: [any]) => {
     return plugins.map((plugin) => {
-      const showCustomIcon = shouldShowCustomIcon(plugin, pluginsMap);
+      const id = `plugin-${plugin.displayName}-${plugin.type}`;
+      const label = plugin.displayName || plugin.name;
       return (
-        <div
-          role="button"
-          className={`plugin-item ${plugin.nodeClass ? plugin.nodeClass : ''} ${
-            plugin.hovering ? 'hovered' : ''
-          }`}
-          // need to implement my popover
-          // my-popover
-          // data-placement="right"
-          // data-template={plugin.template}
-          // content-data={plugin}
-          // data-popover-context={'MySidePanel'} // whole controller
-          // end plugin stuff
-          onClick={(event) => handleClickPlugin(event, plugin)}
-          data-cy={`plugin-${plugin.name}-${plugin.type}`}
+        <Tooltip
+          title={
+            <TooltipText>
+              <span>{label}</span>
+              <br />
+              <br />
```

Comment:
Nit: What do you think about using vertical padding for this type of layout?

Reply:
That makes sense too - i think I'm just going to leave it for now and modify it when we eventually do a design review 👍